### PR TITLE
Fixed bug when removing power generator

### PR DESCRIPTION
**Pull request by Grenkin1988**
Fixed bug after refactoring made on #484
Bug related to #460 and #434

### CODE REVIEW

**Comment by Grenkin1988** [comment]
@TheRiven This should fix #460
@RodRitter This is fixing after #484

File: Assets/Scripts/Models/Furniture.cs [hunk 1]
```diff
@@ -84,6 +84,8 @@ public ENTERABILITY IsEnterable()
     // If this furniture generates power then powerValue will be positive, if it consumer power then it will be negative
     public float powerValue;
 
+    public bool IsPoerConsumer { get { return powerValue < 0; } }
```
Comment (MANDAL0R3) [inline Assets/Scripts/Models/Furniture.cs]: I'm pretty sure there is a typo here ;/

Comment (Grenkin1988) [inline Assets/Scripts/Models/Furniture.cs]: @MANDAL0R3 No it is not :-)

Thanks

**Comment by RodRitter** [comment]
Looks good. Merging.

**Pull request merged**